content/concepts/basics/07-settings.md

```diff
@@ -43,6 +43,6 @@ You have two options here:
 - edit the whole text using the OData Upload action
 - send a POST or PATCH OData request to create or modify settings
 
-> Since settings files are in JSON format, the properties are exposed as regular fields of the settings content. This lets you access and even modify setting values directly through OData without having to edit the text.
+> Since settings files are in json format, the properties are exposed as regular fields of the settings content. This lets you access and even modify setting values directly through OData without having to edit the text.
```

Comment:
```suggestion
> Since settings files are in JSON format, the properties are exposed as regular fields of the settings content. This lets you access and even modify setting values directly through OData without having to edit the text.
```